<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 233, 233, 255), stop:1 rgba(255, 255, 255, 255))}</string>
     </property>
     <widget class="QPushButton" name="load_button">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>460</y>
        <width>104</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(255, 248, 225); /* 奶油白 */
    border: none;
    border-radius: 20px;
    padding: 10px 20px;
    color: #333;
    font-size: 16px;
}

QPushButton:hover {
    background-color: rgb(255, 230, 180); /* Hover 橙黄 */
}

QPushButton:pressed {
    background-color: rgb(255, 215, 140); /* 按下深橙 */
}
</string>
      </property>
      <property name="text">
       <string>上传图片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>361</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/pngtree-cute-cartoon-character-tangerine-png-image_14546254.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>30</y>
        <width>1000</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 35pt &quot;字魂白鸽天行体(商用需授权)&quot;;color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 156, 156, 255), stop:1 rgba(255, 203, 166, 255))</string>
      </property>
      <property name="text">
       <string>水果新鲜程度分析及热量计算系统</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>211</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,9,3">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 156, 156, 255), stop:1 rgba(255, 203, 166, 255));font:20pt &quot;宋体&quot;;background-color:		#FFFAFA;    border-radius: 10px;            /* 圆角半径 */
    padding: 10px;</string>
         </property>
         <property name="text">
          <string>新鲜程度分析</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>3ds Light</family>
           <pointsize>14</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
    background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */
font: 25 14pt &quot;3ds Light&quot;;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>130</y>
        <width>271</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,9,3">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 156, 156, 255), stop:1 rgba(255, 203, 166, 255));font:20pt &quot;宋体&quot;;background-color:		#FFFAFA;    border-radius: 10px;            /* 圆角半径 */
    padding: 10px;</string>
         </property>
         <property name="text">
          <string>请输入体重和性别</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,0,5,0">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 25 10
pt &quot;3ds Light&quot;;</string>
           </property>
           <property name="text">
            <string>体重(kg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">font: 25 10pt &quot;3ds Light&quot;;</string>
           </property>
           <property name="text">
            <string>性别</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;3ds Light&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">
   background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */
font: 25 14pt &quot;3ds Light&quot;;

</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>899</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;宋体&quot;;
    background-color: 	#FFFAFA;      /* 奶油白背景 */
    border-radius: 20px;            /* 圆角半径 */
    padding: 10px;                  /* 内边距让内容别贴边 */

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
